src/compiler/output-targets/dist-lazy/generate-esm-browser.ts

```diff
@@ -24,7 +24,8 @@ export const generateEsmBrowser = async (
       preferConst: true,
       sourcemap: config.sourceMap,
     };
-    if (config.extras.dynamicImportShim) {
+    // TODO(STENCIL-661): Remove code related to the dynamic import shim
+    if (config.extras.__deprecated__dynamicImportShim) {
```

Comment:
I _think_ that `src/utils/util.ts#getDynamicImportFunction` can be marked as deprecated too. You probably saw this as a part of your digging/tracing here, but can we add an explicit TODO (STENCIL-661) to remove it as well? It has 2 callers, here and in https://github.com/ionic-team/stencil/pull/3895/files#diff-5e031103fd3832f45d5667ca75259450d5e42d1fe24dfb7aabd0bd6c143e038fR88 (whose caller you also marked to remove) - I just don't want to forget about this, since there's gonna be a lot of moving pieces 😆

Reply:
yeah it looks like the only codepaths in which it will ever get called are when `dynamicImportShim` is `true`

Reply:
And I agree, it's better for a ticket like STENCIL-661 to be really straightforward and basically a checklist of 'delete all this'